src/superstring/core.clj

```diff
@@ -713,3 +713,173 @@
   {:pre [(string? s)]
    :post (string? %)}
   (java.util.regex.Pattern/quote s))
+
+; URL encoding functions taken from https://github.com/slipstream/SlipStreamUI/blob/master/clj/src/slipstream/ui/util/clojure.clj
```

Comment:
Sorry, indeed. [PR is still open](https://github.com/slipstream/SlipStreamUI/pull/611/files#diff-43997e89f2f4504024e8791a3320af90).